src/pages/SignIn.js

```diff
@@ -1,29 +1,71 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { useState, useEffect, useContext } from'react';
+import { useContext } from'react';
 import {AuthContext} from "../context/AuthContext";
+import InputField from "../components/InputField";
+import {useForm} from "react-hook-form";
+import axios from "axios";
+
 
 
 
 function SignIn() {
-    const {login, logout, isAuth} = useContext(AuthContext);
+    const {login, logout, auth} = useContext(AuthContext);
+    const {register, formState: {errors}, handleSubmit} = useForm();
+
 
     function clickHandler() {
-        isAuth? logout(): login();
+       if (auth.isAuth === true) {
+           logout()
+       }
```

Comment:
In principe kan het geen kwaad om hoe dan ook de logout functie aan te roepen
```suggestion
       logout();
```